File: Q1/Prog52a-Multilingual/main.cpp
#include <iostream>
using namespace std;

int main() {

  int len;

  int wid;
  
  cout << "enter yo length: ";

  cin >> len;

  cout << "";

  cout << "alr cool now enter yo width: ";

  cin >> wid;

  cout << "Your area is: " << (len * wid) << "\n";
  cout << "Your perimeter is: " << ((len*2) + (wid*2));
  
  return 0;
}

File: Q1/Prog52a-Multilingual/main.dart
import 'dart:io';

void main() {
  print('gimmie yo length or the baby gets it: ');
  int? len = int.parse(stdin.readLineSync()!);

  print('alr cool now how about that width: ');
  int? wid = int.parse(stdin.readLineSync()!);

  int area = len * wid;
  int perim = ((len * 2) + (wid * 2));

  print('your area is: ' + area.toString());
  print('your perimeter is: ' + perim.toString());
}
